frontend/src/component/feature/FeatureView/FeatureOverview/FeatureOverviewEnvironments/FeatureOverviewEnvironment/EnvironmentHeader/EnvironmentHeader.tsx

```diff
@@ -17,9 +17,19 @@ const StyledAccordionSummary = styled(AccordionSummary, {
     padding: theme.spacing(0.5, 3, 0.5, 2),
     display: 'flex',
     alignItems: 'center',
+    borderRadius: theme.shape.borderRadiusLarge,
+    pointerEvents: 'auto',
+    opacity: 1,
```

Comment:
I moved these from the parent. Not sure why they were there in the first place? Besides, the way they were added (using the `MuiAccordionSummary--root` class), they'd also target any sub-accordion summaries (such as for release plans). This seems more sensible to me because I'd guess that this is what they're targeting specifically.